.github/workflows/releases.yml

```diff
@@ -0,0 +1,40 @@
+name: Releases
+
+on: push
```

Comment:
Shall we trigger this only on releases?
One can match a branch by its name or a regex here iirc